src/components/CEnqueue.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="container mx-auto py-10 w-3/4">
+    <div class="errors-box">
+      <p v-if="errors.length">
+        <b>Please correct the following error(s):</b>
+        <ul>
+          <li v-for="(error, index) in errors" :key="index">{{ error }}</li>
+        </ul>
+      </p>
+    </div>
+    <form
+      class="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 flex flex-col my-2"
+      @submit.prevent="submit"
+    >
+      <div class="flex flex-wrap -mx-3 mb-6">
+        <div class="w-full md:w-1/2 px-3 mb-6 md:mb-0">
+          <label
+            class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+            for="field_actor_name"
+          >
+            Actor Name
+          </label>
+          <div class="relative">
+            <select
+              class="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+              id="field_actor_name"
+              v-model="message.actorName"
+            >
+              <option
+                v-for="(actor, index) in declaredActors"
+                :key="index"
+                :value="actor"
+              >
+                {{ actor }}
+              </option>
+            </select>
+          </div>
+        </div>
+        <div class="w-full md:w-1/2 px-3">
+          <label
+            class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+            for="grid-last-name"
+          >
+            delay(min)
+          </label>
+          <input
+            class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+            id="grid-last-name"
+            v-model="message.delay"
+            type="number"
+            min="0"
+          />
+          </div>
+        </div>
+        <div class="flex flex-wrap -mx-3 mb-2">
+          <div class="w-full md:w-1/3 px-3 mb-6 md:mb-0">
+            <label
+              class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+              for="field_args"
+            >
+              Args
+            </label>
+            <textarea
+              class="appearance-none  h-40 block w-full bg-gray-200 focus:bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:border-gray-500"
+              id="field_args"
+              placeholder='[1, ["a", 7], "str"]'
+              v-model="message.args"
+            ></textarea>
+          </div>
+
+          <div class="w-full md:w-1/3 px-3 mb-6 md:mb-0">
+            <label
+              class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+              for="field_kwargs"
+            >
+              Kwargs
+            </label>
+            <textarea
```

Comment:
What would be nice is to check on unfocus that the field is valid (Json) and have a red ouline when in error with a message next to the label.